Bottom: ca95bafdf74ce1818fb79ea0e1828bdd2b019d5d
Top:    543a5d8863f546b742cf34435f7e1dd649ef9381
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-31 16:12:29 -0700

Refresh of addrtype

---

diff --git a/examples/cmatose.c b/examples/cmatose.c
index 167f509..8ca2d88 100644
--- a/examples/cmatose.c
+++ b/examples/cmatose.c
@@ -390,7 +390,7 @@ static int alloc_nodes(void)
 		if (dst_addr) {
 			ret = rdma_create_id(test.channel,
 					     &test.nodes[i].cma_id,
-					     &test.nodes[i], RDMA_PS_TCP);
+					     &test.nodes[i], hints.ai_port_space);
 			if (ret)
 				goto err;
 		}
@@ -505,7 +505,7 @@ static int run_server(void)
 	int i, ret;
 
 	printf("cmatose: starting server\n");
-	ret = rdma_create_id(test.channel, &listen_id, &test, RDMA_PS_TCP);
+	ret = rdma_create_id(test.channel, &listen_id, &test, hints.ai_port_space);
 	if (ret) {
 		perror("cmatose: listen request failed");
 		return ret;
